src/main/java/uk/gov/hmcts/reform/prl/services/hearingmanagement/HearingManagementService.java

```diff
@@ -88,35 +87,26 @@ public void caseStateChangeForHearingManagement(HearingRequest hearingRequest) t
             authTokenGenerator.generate(),
             hearingRequest.getCaseRef()
         );
+
         CaseData caseData = CaseUtils.getCaseData(caseDetails, objectMapper);
 
-        String hmcStatus = hearingRequest.getHearingUpdate().getHmcStatus();
-        switch (hmcStatus) {
-            case LISTED:
+
+        switch (caseState) {
```

Comment:
do we still need a switch case when we are passing the state from fis-hmc-api to prl-cos-api

Reply:
Still we need it as we may expect two kind of Case State such as DECISION_OUTCOME, PREPARE_FOR_HEARING. but in the future, there could be more. So instead of choosing IF ELSE, I have gone with SWITCH